source-academy/frontend
# Integration of the Explicit Control Evaluator with the current Environmental Visualiser

### Description

This PR allows the current environment visualizer to run using the explicit control evaluator as intended. It is required for the changes introduced by [source-academy/js-slang/pull/1361](https://github.com/source-academy/js-slang/pull/1361)  to work as intended. Until [source-academy/js-slang/pull/1361](https://github.com/source-academy/js-slang/pull/1361) is not merged, and the js-slang version is not bumped, this PR does not change any current functionality.

### Type of change


### How to test
Checkout js-slang to https://github.com/source-academy/js-slang/pull/1361, and then [link local js-slang to local frontend](https://github.com/source-academy/js-slang#using-your-js-slang-in-your-local-source-academy), and start the frontend locally. The environment visualizer should now be using the explicit control evaluator instead of the interpreter.
